jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletChannelState.java

```diff
@@ -334,15 +329,37 @@ public boolean abortResponse()
                 case OPEN:
                     _servletChannel.getServletContextResponse().setStatus(HttpStatus.INTERNAL_SERVER_ERROR_500);
```

Comment:
This feels arbitrary and in the wrong spot.

Reply:
Removed.